idp-core/src/main/java/io/mosip/idp/core/dto/AuditableIdPTransaction.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+package io.mosip.idp.core.dto;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+public class AuditableIdPTransaction {
```

Comment:
As discussed lets rename this class to just AuditDTO

Reply:
Done